upup/pkg/fi/cloud.go

```diff
@@ -27,6 +27,23 @@ type Cloud interface {
 	ProviderID() CloudProviderID
 
 	DNS() (dnsprovider.Interface, error)
+
+	// FindVPCInfo looks up the specified VPC by id, returning info if found, otherwise (nil, nil)
+	FindVPCInfo(id string) (*VPCInfo, error)
+}
+
+type VPCInfo struct {
```

Comment:
Since we are pulling other AWS structs into memory - can we at least add a pointer here to the raw data? In case we need it for anything else?